spec/01-unit/01-db/01-schema/06-routes_spec.lua

```diff
@@ -1512,6 +1512,7 @@ describe("routes schema (flavor = expressions)", function()
   end)
 
   it("http route still supports net.port but with warning", function()
+    local log = spy.on(ngx, "log")
```

Comment:
Remember to recover the `ngx.log` after the test. Use setup and teardown to make sure it won't affect other tests.

Reply:
Please check the `finally` to ensure I'm corret.

Reply:
I got an error `spec/01-unit/01-db/01-schema/06-routes_spec.lua:1519:7: setting read-only field log of global ngx`,
What is the right way to this?